### PR TITLE
Issue #263 page base type issue

### PR DESCRIPTION
**Pull request by mason-chase**
I'm trying to replicate the issue here

### CODE REVIEW

File: Samples/WebFormsSample/BasePage.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿// MIT License.
+
+using System.Web.UI;
```
Comment (twsouthwick) [inline Samples/WebFormsSample/BasePage.cs:3]: no need to have a sample and a test with this. the test is probably a better location so we detect regressions

Comment (mason-chase) [reply Samples/WebFormsSample/BasePage.cs:3]: It wasn't clear for me what should I do with this? move it to another separate test?
The idea of this PR at this moment is to replicate the issue and hopefully get some help on what is the solution to fix it?  

I'm currently blocked on this.

Comment (twsouthwick) [reply Samples/WebFormsSample/BasePage.cs:3]: Gotcha - I'll clone it later today and take a look

Comment (mason-chase) [reply Samples/WebFormsSample/BasePage.cs:3]: Let me know if that worked for you.

**Comment by twsouthwick** [comment]
Closing in favor of #317

**Pull request closed without merging**